VRCFaceTracking/Services/ActivationService.cs

```diff
@@ -116,7 +116,20 @@ private async Task StartupAsync()
         _logger.LogInformation("Checking for updates for installed modules...");
         var localModules = _moduleDataService.GetInstalledModules().Where(m => m.ModuleId != Guid.Empty);
         var remoteModules = await _moduleDataService.GetRemoteModules();
-        var outdatedModules = remoteModules.Where(rm => localModules.Any(lm => rm.ModuleId == lm.ModuleId && rm.Version != lm.Version));
+        var outdatedModules = remoteModules.Where(rm => localModules.Any(lm =>
+        {
+            if (rm.ModuleId != lm.ModuleId) 
+                return false;
+
+            var remoteVersion = new Version(rm.Version);
+            var localVersion = new Version(lm.Version);
+
+            var result = remoteVersion.CompareTo(localVersion);
+            if (result > 0) // remote module version is higher than registry
+                return true;
+
+            return false;
```

Comment:
Could this be simplified to `return remoteVersion.CompareTo(localVersion) > 0;`?

Reply:
Sure